puncover/puncover.py

```diff
@@ -52,7 +52,10 @@ def open_browser(host, port):
 
 
 def main():
-    gcc_tools_base = os.path.join(find_arm_tools_location(), 'bin/arm-none-eabi-')
+    try:
+        gcc_tools_base = os.path.join(find_arm_tools_location(), 'bin/arm-none-eabi-')
+    except TypeError:
+        gcc_tools_base = None
```

Comment:
I think the better solution looks like the following:
```python
arm_tools_location = find_arm_tools_location()
gcc_tools_base = os.path.join(arm_tools_location, 'bin/arm-none-eabi-') if arm_tools_location is not None else None
```

Reply:
@katyo I guess the end result is the same, I don't have any opinion on which way is better, I'll leave it to @HBehrens to decide

Reply:
As I think, a better and quick way to fix this is returnning `'/usr'` instead of `None` from `find_arm_tools_location()` function since most *nixes install `arm-none-eabi-objdump` under `/usr/bin`.

Reply:
This should be in form `<path>/<pfx>-`.